### PR TITLE
Update razers3

### PR DESCRIPTION
**Pull request by FelixMoelder**
Updates razers3 to v3.5.8.
Additionally, adds binary for macOS.

### CODE REVIEW

**Comment by FelixMoelder** [comment]
@bioconda-bot please add labels

File: recipes/razers3/meta.yaml [hunk 1]
```diff
@@ -1,25 +1,17 @@
...
+
 package:
   name: razers3
-  version: '3.5.3'
+  version: {{ version }}
 
 source:
-  url: http://packages.seqan.de/seqan-src/seqan-src-2.1.1.tar.gz
-  sha256: df083d5939d7e73fd00bbc6bb291ac7ea7b8d446cf89a9758fdffdc0eca9ec1c
+  - url:  http://packages.seqan.de/razers3/razers3-{{ version }}-Mac-x86_64.zip                 # [osx]
```
Comment (bgruening) [inline recipes/razers3/meta.yaml:8]: @FelixMoelder we do prefer to compile from source, it that possible?

Comment (FelixMoelder) [reply recipes/razers3/meta.yaml:8]: I tried that before.
But as the VM comes with Xcode 9 it tries to build for older macOS versions (below macOS 12) which are not supported.

Comment (FelixMoelder) [reply recipes/razers3/meta.yaml]: What works is to build the linux binary from source as before and use the precompiled binary for macOS.

Comment (dpryan79) [reply recipes/razers3/meta.yaml]: Xref: https://github.com/bioconda/bioconda-recipes/pull/21180

**Comment by FelixMoelder** [comment]
@bioconda-bot please merge

**Comment by mbargull** [comment]
@BiocondaBot, please fetch artifacts

**Comment by mbargull** [comment]
Nice that it worked out in the end! I'm just confused why the `conda_build_config.yaml` route didn't work ¯\\\_(ツ)\_/¯.

**Pull request merged**